td-shim-tools/Cargo.toml

```diff
@@ -62,7 +62,7 @@ parse_int = { version = "0.6.0", optional = true }
 
 [features]
 default = ["boot-kernel", "enroller", "linker", "signer", "loader", "tee"]
-boot-kernel = ["td-layout/boot-kernel", "td-shim/boot-kernel"]
+boot-kernel = []
```

Comment:
why we still keep it?

Reply:
`td-shim-tools` still keeps the feature, will remove it in another PR and use input parameter instead